portal/component-tests/contract/components/abandon-deal-link.component-test.js

```diff
@@ -4,9 +4,17 @@ const component = 'contract/components/abandon-deal-link.njk';
 const render = componentRenderer(component);
 
 describe(component, () => {
-  describe('when viewed by a maker', () => {
+  const makerRole = ['maker'];
+  const nonMakerRoles = ['checker', 'admin', 'read-only'];
+
+  function makerRoleTests(roleToCombineWithMaker) {
```

Comment:
We probably don't need the roleToCombineWithMaker anymore

Reply:
We could land up having a `maker` with combined role of a `checker` or and `admin`.